delivery_correos_express/models/correos_express_request.py

```diff
@@ -68,7 +67,9 @@ def _send_api_request(self, request_type, url, data=None, skip_auth=False):
                 res = requests.post(url=url, auth=auth, json=data, timeout=60)
             else:
                 raise UserError(
-                    _("Unsupported request type, please only use 'GET' or 'POST'")
+                    self.carrier_id.env._(
```

Comment:
And the same for the rest:

```suggestion
                    self.env._(
```

Reply:
The carrier_id is necessary since the class is a normal python class and has no env